osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -330,6 +330,21 @@ private void load(OsuColour colours, LocalisationManager localisation, BeatmapDi
                 addInfoLabels();
             }
 
+            protected override void LoadComplete()
+            {
+                base.LoadComplete();
+
+                mods.BindValueChanged(m =>
+                {
+                    settingChangeTracker?.Dispose();
+
+                    refreshBPMLabel();
```

Comment:
As seen in test failures, `refreshBPMLabel()` unconditionally accesses `bpmLabelContainer` which isn't necessarily created in BDL due to this early-return:

https://github.com/ppy/osu/blob/35a208a478f11b6446f29a91453aa4f65bab0372/osu.Game/Screens/Select/BeatmapInfoWedge.cs#L341-L342

A couple of ways to fix are available, not sure which is the best - maybe enough to add a null-check for that container inside of `refreshBPMLabel()` and call it a day?

Reply:
I had a null check. It must have been lost during commit 😨.